translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionResultAssertion.kt

```diff
@@ -0,0 +1,12 @@
+package ch.tutteli.atrium.translations
+
+import ch.tutteli.atrium.assertions.DescriptiveAssertion
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+
+/**
+ * Contains the [DescriptiveAssertion.description]s of the assertion functions which are applicable to [Map].
+ */
+enum class DescriptionResultAssertion(override val value: String) : StringBasedTranslatable {
+    VALUE("value"),
+    ILLEGAL_STATE_EXCEPTION("Illegal state exception !!")
```

Comment:
remove ILLEGAL_STATE_EXCEPTION and re-add `IS_NOT_SUCCESS("!! is not a Success"),` as you had earlier